WordPressVIPMinimum/Sniffs/Sniff.php

```diff
@@ -17,4 +17,45 @@
  * @package VIPCS\WordPressVIPMinimum
  */
 abstract class Sniff extends \WordPressCS\WordPress\Sniff {
+	/**
```

Comment:
Yes, the method in WPCS is now marked as not part of the "public API", but it can still be _used_.

The "not part of the public API", just means "use at your own risk as there may be BC-breaking changes made in non-major releases", but until such a time that such a BC-breaking change would be made, VIPCS could still use it as the functionality hasn't changed (at this time and I don't currently have any plans to change it either).

So based on that, I have two questions:
1. Why copy this method ?
2. If copying is really the way to go, why not follow the example set by WPCS and have it in a separate `Helper` class instead of adding it to the `Sniff` class ?
    Having it in a separate `Helper` class, will allow for the method to still be available to sniffs if any of them switch over to `abstract` base classes from PHPCSUtils in the future (which is exactly the reason why it was moved to a `Helper` in WPCS).

Reply:
Opened #744 to handle separately because all of the rest of this PR can then be framed as "Make some use of PHPCSUtils", rather than "Be compatible with WPCS 3.0".